2200075/5_1225 . 암호생성기.py

```diff
@@ -0,0 +1,21 @@
+from collections import deque
+
+T = 10
+for t in range(T) :
+    nothing = input()
+    numbers = deque((map(int, input().split())))
```

Comment:
덱 활용 아주 좋습니다! 👍 
```suggestion
    numbers = deque(map(int, input().split()))
```
불필요한 괄호는 지워주세요 😄